test/phx_tools_web/live/phx_tools_landing_live_test.exs

```diff
@@ -0,0 +1,38 @@
+defmodule PhxToolsWeb.PhxToolsLandingLiveTest do
```

Comment:
Name does not match the name of the module being tested.
The convention is to use the module name and put "Test" at the end.